orchestrator-salt/src/main/resources/salt/salt/disks/mount/scripts/find-device-and-format.j2

```diff
@@ -104,6 +104,17 @@ save_env_vars_to_log_file() {
     log $LOG_FILE CLOUD_PLATFORM=$CLOUD_PLATFORM
 }
 
+get_temporary_disk_device_name() {
+  if [[ $CLOUD_PLATFORM -eq AZURE ]]; then
+    temporary_disk="/dev/$(ls -l /dev/disk/azure/ | awk '{print $9"\t"$11}' | grep -E '(resource[^-])' | cut -f2 | cut -f3 -d'/')"
```

Comment:
Question: what happens if there is no temporary disk? I think then the value of temporary_disk would be "/dev".
On AWS the return value is empty if there are no ephemeral disks. Will this difference cause no problems in the mounting? Will the script not want to mount the "/dev"?